Sources/Sentry/SentryClient.m

```diff
@@ -468,17 +469,6 @@ - (SentryId *)sendEvent:(SentryEvent *)event
     return preparedEvent.eventId;
 }
 
-- (NSArray *)attachmentsForEvent:(SentryEvent *)event scope:(SentryScope *)scope
```

Comment:
I pushed this method down close to `addAttachmentProcessor` and `removeAttachmentProcessor`.